pkg/compiler/codegen.go

```diff
@@ -513,6 +521,11 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 			// Use the ident to check, builtins are not in func scopes.
 			// We can be sure builtins are of type *ast.Ident.
 			c.convertBuiltin(n)
+
+			if isAppCall {
+				buf := c.getByteArray(n.Args[0])
+				c.prog.WriteBytes(buf)
```

Comment:
So if `getByteArray` returns `nil` it will happily crash?

Reply:
I have added a compile error if the length is less than 20.